.circleci/config.yml

```diff
@@ -3,16 +3,26 @@ version: 2
 jobs:
   build:
     docker:
-      - image: docker:17.05.0-ce-git
+      - image: circleci/python:3.6.1
```

Comment:
Since the previous image was `docker` and this is a repo of Dockerfiles, perhaps its worth a comment documenting why we're using the python image and what we expect it to provide (bash, docker, python3).